_includes/releases/v23.1/v23.1.0-alpha.3.md

```diff
@@ -0,0 +1,102 @@
+
+<h3 id="v23-1-0-alpha-3-sql-language-changes">SQL language changes</h3>
+
+- Added [latency info](https://www.cockroachlabs.com/docs/cockroachcloud/statements-page.html#statement-statistics) (min, max, p50, p90, p99) to crdb_internal.statement_statistics, system.statement_statistics and crdb_internal.cluster_statement_statistics. Also adds `latency_seconds_min`, `latency_seconds_max`, `latency_seconds_p50`, `latency_seconds_p90` and `latency_seconds_p99` to crdb_internal.node_statement_statistics. [#96396][#96396]
+- Deprecated the PGDUMP and MYSQLDUMP formats for [IMPORT](https://www.cockroachlabs.com/docs/stable/import-performance-best-practices.html). They are still present, but will be removed in a future release. View this [page](https://www.cockroachlabs.com/docs/v22.2/migration-overview) for alternatives. [#96386][#96386]
+- [`COPY ... FROM ... QUOTE '"'`](https://www.cockroachlabs.com/docs/v22.2/copy-from) will no longer error. [#96572][#96572]
+- Added `last_error_code` column to the `crdb_internal.node_statement_statistics` table. Added `last_error_code` field to the `statistics` JSON blob in the `crdb_internal.statement_statistics` and `system.statement_statistics` tables. [#96436][#96436]
+- Added support for expressions of the form `COLLATE "default"`, `COLLATE "C"`, and `COLLATE "POSIX"`. Since the default [collation](https://www.cockroachlabs.com/docs/stable/collate.html) cannot be changed currently, these expressions are all equivalent. The expressions are evaluated by treating the input as a normal string, and ignoring the collation.  This means that comparisons between strings and collated strings that use "default", "C", or "POSIX" are now supported.  Creating a column with the "C" or "POSIX" collations is still not supported. [#96828][#96828]
+- The insights subsystem in `sqlstats` is now able to detect failed executions, regardless if they were slow or not. [#97039][#97039]
+- The internal `statement_statistics` and `transaction_statistics` tables now include sampled execution statistics on storage iteration. [#96016][#96016]
+- Introduced the `declare_cursor_statement_timeout_enabled` session variable which disables statement timeouts during FETCH when using DECLARE CURSOR. [#96607][#96607]
+
+<h3 id="v23-1-0-alpha-3-operational-changes">Operational changes</h3>
+
+- A [BACKUP](https://www.cockroachlabs.com/docs/stable/backup.html) which encounters too many retryable errors will now fail instead of pausing to allow subsequent backups the chance to succeed. [#96673][#96673]
+- Added option to balance CPU time (cpu) instead of queries per second (qps) among stores in a cluster. This is done by setting `kv.allocator.load_based_rebalancing.objective='cpu'`. `kv.allocator.cpu_rebalance_threshold` is also added, similar to `kv.allocator.qps_rebalance_threshold` to control the target range for store cpu above and below the cluster mean. [#96127][#96127]
```

Comment:
```suggestion
- Added an option to balance CPU time (`cpu`) instead of queries per second (`qps`) among stores in a cluster. This is done by setting `kv.allocator.load_based_rebalancing.objective='cpu'`. `kv.allocator.cpu_rebalance_threshold` is also added, similar to `kv.allocator.qps_rebalance_threshold` to control the target range for store CPU above and below the cluster mean. [#96127][#96127]
```